### PR TITLE
fix: read the rows to assign maximum and minimum to the filters

### PR DESCRIPTION
**Pull request by AlbaLopez**

### CODE REVIEW

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts [hunk 1]
```diff
@@ -137,9 +126,11 @@ export class JobResultTableComponent implements AfterViewInit, OnInit, OnDestroy
     private store: Store<AppState>,
     private router: Router,
     private dialog: MatDialog,
-  ) {}
+  ) {
+  }
```
Comment (kvikshaug) [inline src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts:130]: There are some style changes like this (changed indentation, spaces etc), it would be nice to add those in separate commits so it's easier to filter on the actual changes.

**Review by kvikshaug — COMMENTED** [review]
Since the logic is run in `ngOnInit`, what happens if you open the component, navigate back, and then open a different job? Will the min/max values use values for the previous job, or will they update?

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts (hunk 1 above)
Comment (AlbaLopez) [reply src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts:130]: ngOnInit is executed every time that you go to the component, so the values will change :)
